app/lib/submission_builder/return_w2.rb

```diff
@@ -24,11 +24,10 @@ def document
           stpickup_node.add_child(desc_node)
           stpickup_node.add_child(amt_node)
 
-          existing_deductions = xml_node.at(:OtherDeductionsBenefitsGrp)
-          if existing_deductions
-            existing_deductions.add_next_sibling(stpickup_node)
-          else
+          if state_local_tax_grp_node.present?
             state_local_tax_grp_node.add_previous_sibling(stpickup_node)
+          else
+            xml_node.at(:StandardOrNonStandardCd)&.add_previous_sibling(stpickup_node)
```

Comment:
I feel like this may hide the error if we end up not being able to add the sibling correctly -- can we remove the safe navigation here?